doc/linters.md

```diff
@@ -286,10 +286,48 @@ Depending on undefined task: compile
          b {:depends [a]}}}
 ```
 
+### Bb.edn global requires
+
+*Keyword:* `:bb.edn-global-requires`
+
+*Description:* warn on top level `:requires` key.
+
+*Default level:* `:warning`
+
+*Example trigger:*
+
+`bb.edn`:
+
+``` clojure
+{:requires [[babashka.fs :as fs]]}
+```
+
+*Example message:*
+
+```
+Global :requires belong in the :tasks map.
+```
+
+### Bb.edn task docstring missing
+
+*Keyword:* `:bb.edn-task-docstring-missing`
```

Comment:
`bb.edn-task-missing-docstring` would me more consistent with the other `missing-docstring`

Reply:
I think this warning should be disabled by default for "private" tasks (the task name starts with `-` or has `:private true`).

Reply:
Good point. It seems the missing-docstring linter also warns for private functions, so let's just keep the behavior as you have it now.

Reply:
we could reverse it and have it complain for private by default. that way it's consistent with missing-docstring but could also be customized further?

Reply:
fwiw i'm halfway there already :)

Reply:
> we could reverse it and have it complain for private by default. that way it's consistent with missing-docstring

Let's just do that, for consistency and leave it at that for now.

Reply:
ok, so just added test cases for private tasks as well so the behaviour is covered